ncm-network/src/main/perl/network.pm

```diff
@@ -1481,10 +1489,30 @@ sub stop
     if ($nwupdated || @ifaces) {
         if (@ifaces) {
             my @cmds;
+            # delete NetworkManager connections
+            foreach my $file ( glob "/etc/NetworkManager/system-connections/*.nmconnection" ) {
```

Comment:
i don't know how networkmanager works, but ncm-network will try it's best to restore previous working network if the component would hit some error.
if these files are required for such recover, you best not delete them; but eg rename and make sure they get restored if needed (and cleaned up when working and really not needed anymore)

Reply:
Any automatically (during install) or manually created files in that path will take precedence, they are deleted so the legacy 'ifcfg' files are used by NetworkManager instead, the backup/recovery will continue to work.

Reply:
not if this is the initial install, then recovery will be broken without these files.

Reply:
OK, I'll try to add these files to the recovery instead